docuploader/__main__.py

```diff
@@ -89,7 +89,16 @@ def upload(
 
     metadata = metadata_pb2.Metadata()
     text_format.Merge(metadata_path.read_text(), metadata)
-    # TODO: Validate that the minimum metadata is set.
+
+    # Validating metadata for required fields
+
+    # TODO: Do additional validiation
+    if not metadata.name:
```

Comment:
Is there a better way to check for a field?

Reply:
This is fine ("Falsy" covers `None` and `""`).